### DISCUSSION TITLE
Question about latency calculation

### DISCUSSION THREAD

**Discussion by MilleBo** [VIP-Logic/NBomber#506] [Q&A]
I recently started to look at this framework to do some minor performance testing on our API and have a quick question about latency calculation. 

I'm currently resuing a bit of my old client/services and those already have some time logging since before (by doing internal stopwatch start/stop before the API calls). And if I put these client/services in differnt steps and scenarios and run NBomber I can see that the final latency report pretty much follow the result I get from my old time checks (NBomber usually have a bit higher times but I guess that's expected as it takes the time for the whole step and my stop watches is only the actual API call).

But the issue is that sometimes I get totally different results between my internal time checks and NBomber. For exampel NBomber can say that the min latecy was 500 ms while my internal timer say that the min was 1.5 seconds. 

So the question is if I don't really understand how the latecy calculation work? I thought it was the whole execution time of a step or is it something more behind it? Because I expected that my internal time checks would always be lower but as I said I sometimes get shorter times fron NBomber.

**Comment by AntyaDev** [comment]
Hi @MilleBo 

Can you try the latest v4 beta?
